src/main/scala/org.rogach.scallop/DefaultConverters.scala

```diff
@@ -37,16 +37,18 @@ trait DefaultConverters {
   }
 
   // for some reason direct usages of singleArgConverter sometimes fail to compile under Dotty 0.27.0-RC1
-  // this wrapper "fixes" the issue
+  // this wrapper "fixes" the issue.
+  // __Note:__ this was due to package object scallop using the default argument for java.io.File first;
+  // Explicit handler is now defined, and this method should be superfluous.
```

Comment:
Great, I'll remove it then.

Reply:
I didn't do it yet because technically it would be binary incompatible. Perhaps just put `@deprecated` and remove it in the next major version?

Reply:
Well, the next version will be the next major version (there's a small slightly-breaking behavior change due to modifications for #147, which is about to land). So we can remove it right now, and I'll include a note in the changelog just in case (should have made it private in the first place - but even if somebody used this then the fix should be quite simple).